src/_help/hubs/hub-settings.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Hub settings
+---
+
+- TOC
```

Comment:
I'm sorry as I might "join the battle when it's over". So if that's the case, please discard the following.

I'm surprised by the structure of the page. My expectations would be that I get the description of each of the tabs, and each of the settings and actions that I can play within each of the tabs. So something like:
## Settings
### General settings
### Hub access
### UI customization
### Danger zone
## Access Management
### ...
## Documentation (which should have an "s", in the webapp, btw)

I admit there's some redundancy, especially on the Access Management part, but I'm thinking we're missing some of the things that the user can do, and that the guidance on "where to find what setting or action" could be more transparent.